### PR TITLE
Drink components

### PR DESCRIPTION
**Pull request by arjanmitra**
-created AllVenues component to display as a landing page when the app loads for patron
-created AllDrinks component to display all the drinks of an ACTIVE menu for a patron to view and choose from (incDrink, decDrink hasn't been implemented yet)
-created SingleDrink component to display specific drink
-added some styles for the above
-added some motion attributes for the above

### CODE REVIEW

File: client/components/utils/IncDrink.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const incDrink = async (id) => {
+  try {
+    //check if user token exists, if not, redirect to login/register page. otherwise continue:
+    //check if active tab already exists. if not, create new tab and new order and add on drink. if so, add onto existing order
```
Comment (domisobi) [inline client/components/utils/IncDrink.js:4]: @justinmattos I think here is where we could add the put request that you mentioned

File: client/styles/GlobalStyle.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ const GlobalStyle = createGlobalStyle`
     a {
       color: var(--orange);
     }
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```
Comment (domisobi) [inline client/styles/GlobalStyle.js:21]: we might have this doubled, I can pull mine out

**Review by domisobi — APPROVED** [review]

File: client/styles/GlobalStyle.js (hunk 2 above)
Comment (arjanmitra) [reply client/styles/GlobalStyle.js:21]: i think git automatically merged this, so we should be good!

File: client/components/utils/IncDrink.js (hunk 1 above)
Comment (justinmattos) [reply client/components/utils/IncDrink.js:4]: Indeed, this would be a good place for it

**Pull request merged**